src/vtex-ng-filter.html

```diff
@@ -75,6 +75,11 @@ <h3 class="group-header">
                ng-if="openFilters[filter.rangeUrlTemplate + 'Selector']">
 
             <div class="row-fluid">
+              <!-- Message used in OMS -->
+              <div ng-if="filterTimeLimit">
+                <i class="icon-info-sign icon-blue"></i>&nbsp;
+                <i translate>search.date-range-limit</i>
```

Comment:
Deveria ser uma tag `i` aqui tambem?

Reply:
Shortcut para um texto itálico ja que não tem o tachyons com `.ti`.

Reply:
isso é cheat rs